### ISSUE TITLE
Headers are getting URL encoded?

### ISSUE THREAD

**Issue by kevinrenskers** [borchero/Squid#12]
In my HTTPService I set a header:

```
var header: HttpHeader {
  HttpHeader([
    .accept: HttpMimeType.json.rawValue,
    .authorization: "Bearer \(token)",
  ])
}
```

This is getting sent as `Bearer%20DsNYO6w84ZaJBJyYWHEtStTBoRCquu3XAQc0tVH8ggq3orn7UAYGYvfrU8ZjYKQTHvKB2W28xZuUjMmZ`, and the server really doesn't like this. Why is it getting encoded, and can I do something about that?

**Comment by borchero** [comment]
Ouh that shouldn't happen. I'll have a look what's going on here.

**Comment by borchero** [comment]
Fixed in release 1.3.2. I honestly don't know why the values where escaped in the first place though...

**Comment by kevinrenskers** [comment]
Yea, that seems like something the dev should do in their app code? Anyway, thanks again! :)
